poly-commitment/src/tests/commitment.rs

```diff
@@ -150,8 +150,14 @@ fn test_randomised<RNG: Rng + CryptoRng>(mut rng: &mut RNG) {
 
             let mut chunked_evals = vec![];
             for point in eval_points.clone() {
+                let n = poly.len();
+                let num_chunks = if n % srs.g.len() == 0 {
+                    n / srs.g.len()
+                } else {
+                    n / srs.g.len() + 1
+                };
                 chunked_evals.push(
-                    poly.to_chunked_polynomial(1, srs.g.len())
+                    poly.to_chunked_polynomial(num_chunks, srs.g.len())
```

Comment:
If I understand this correctly, `num_chunks` is always `1` here... So you just generalised it?

Reply:
See https://github.com/o1-labs/proof-systems/pull/2010#discussion_r1529950740